src/projections/index.js

```diff
@@ -6,3 +6,7 @@ export { default as decoration } from './decoration';
 export { default as defaultContent } from './default-content';
 export { default as customRow } from './custom-row';
 export { default as sorting } from './sorting';
+export { default as PolyfillColspan } from './polyfill-colspan';
```

Comment:
Shall we call this `colspan-zero`?

Reply:
I plan to call all postProjection as polyfill-XXX in post projection